variables.tf

```diff
@@ -444,3 +444,21 @@ variable "webhooks" {
   #   insecure_ssl = false
   # }]
 }
+
+variable "secrets" {
+  description = "Configuring actions secrets. For details please check: https://www.terraform.io/docs/providers/github/r/actions_secret.html"
```

Comment:
Adding that `sensitive` field here would make that the minimum terraform version to be used is `0.14.0` , is that ok ?

When running the test, I'm using terraform version `0.13.5`
```
Error: Unsupported argument

  on ../../variables.tf line 450, in variable "plaintext_secrets":
 450:   sensitive   = true

An argument named "sensitive" is not expected here.
}
    destroy.go:11: 
        	Error Trace:	destroy.go:11
        	            				panic.go:617
        	            				testing.go:732
        	            				apply.go:15
        	            				public_repository_with_secrets_test.go:36
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; 
        	            	Error: Unsupported argument
        	            	
        	            	  on ../../variables.tf line 450, in variable "plaintext_secrets":
        	            	 450:   sensitive   = true
        	            	
        	            	An argument named "sensitive" is not expected here.
        	            	}
        	Test:       	TestGithubPublicRepositoryWithSecret
--- FAIL: TestGithubPublicRepositoryWithSecret (0.15s)
```

Reply:
Without adding the `sensitive` field, this is an example of terraform output:
```
  # module.test-repo.github_actions_secret.repository_secret[1] will be created
  + resource "github_actions_secret" "repository_secret" {
      + created_at      = (known after apply)
      + id              = (known after apply)
      + plaintext_value = (sensitive value)
      + repository      = "test-repo"
      + secret_name     = "MY_SECRET"
      + updated_at      = (known after apply)
    }
```
I guess that it gets substitued the secret by `(sensitive value)` because of the github provider, isn't it ? Because in the code there is no variable set as `sensitive = true`

Reply:
yes.. my bad... i was so excited when they added it, I forgot about backward compatibility... so you are right, let's not add it and keep compatibility as long as possible..